packages/siwe/core/utils/AppKitAuthUtil.ts

```diff
@@ -0,0 +1,108 @@
+import { formatMessage } from '@walletconnect/utils'
+import { Web3ModalSIWEClient } from '../../src/client.js'
+import type { SIWECreateMessageArgs, SIWESession, SIWEVerifyMessageArgs } from './TypeUtils.js'
+import { ConstantsUtil } from '@web3modal/common'
+
+const myHeaders = new Headers()
+myHeaders.append('x-project-id', '24970167f11c121f6eb40b558edb9691')
```

Comment:
I guess this is the problem where you wanted to get these values dynamically right?

Reply:
No this is temporary until the API auth routes are live in prod.
This is a Cloud staging project id.